modules/wallet/persist.go

```diff
@@ -14,14 +17,6 @@ type savedKey struct {
 	Visible          bool
 }
 
-// legacySavedKey preserves compatibility with the other beta wallets. After
-// the full currency is launched, this struct and the related code can be
-// discarded.
-type legacySavedKey struct {
-	SecretKey        crypto.SecretKey
-	UnlockConditions types.UnlockConditions
-}
```

Comment:
legacy keys are removed?

I guess it should be okay, but it does seem a bit weird to only support them for 2 weeks.